samples/snippets/csharp/VS_Snippets_CLR_System/system.threading.tasks.taskfactory.startnew/cs/startnew1.cs

```diff
@@ -33,7 +33,7 @@ private static string ShowHex(string value)
       // Handle only non-control characters.
       if (! Char.IsControl(value, 0)) {
          foreach (var ch in value)
-            hexString += String.Format("0x{0} ", Convert.ToUInt16(ch));
+            hexString += $"0x{(ushort)ch} ";

```

Comment:
I made equivalent code like everywhere before, but I feel like we should do `:X` for hex format, otherwise it's misleading, since it's outputting decimal format. Or am I wrong?

Reply:
I would prefer the `:x` as well. That would be more resilient.